definitions/json/EventToComplexTypes/EventToComplexTypes.json

```diff
@@ -698,6 +698,33 @@
     "FieldDisplayOrder": 2,
     "DisplayContext": "MANDATORY"
   },
+  {
+    "ID": "CreateCase",
+    "CaseEventID": "manageCaseTTL",
+    "CaseFieldID": "TTL",
+    "ListElementCode": "SystemTTL",
+    "EventElementLabel": "System TTL",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "READONLY"
+  },
+  {
+    "ID": "CreateCase",
+    "CaseEventID": "manageCaseTTL",
+    "CaseFieldID": "TTL",
+    "ListElementCode": "OverrideTTL",
+    "EventElementLabel": "Override TTL",
+    "FieldDisplayOrder": 2,
+    "DisplayContext": "OPTIONAL"
+  },
+  {
+    "ID": "CreateCase",
+    "CaseEventID": "manageCaseTTL",
+    "CaseFieldID": "TTL",
+    "ListElementCode": "Suspended",
+    "EventElementLabel": "Suspend Deletion",
+    "FieldDisplayOrder": 3,
+    "DisplayContext": "OPTIONAL"
+  },
```

Comment:
Unsure about this because there's no reference to TTL in the CreateCase event in CaseEventToFields

Reply:
Sorted as agreed